Networking/Networking/Model/Refund/LineItemRefund.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/// Represents a Line Item to be Refunded
+///
+public struct LineItemRefund {
+
+    ///  ID from the order's line_items
+    ///
+    public let itemID: String
+
+    /// Item quantity
+    ///
+    public let quantity: Int
+
+    /// Item price without tax
+    ///
+    public let refundTotal: String?
+
+    /// Taxes to refund
+    public let refundTax: [TaxRefund]?
+
+    /// Line Item Refund struct initializer
+    ///
+    public init(itemID: String, quantity: Int, refundTotal: String?, refundTax: [TaxRefund]?) {
+        self.itemID = itemID
+        self.quantity = quantity
+        self.refundTotal = refundTotal
+        self.refundTax = refundTax
+    }
+
+    public func toDictionary() -> [String: Any] {
+        var dict: [String: Any] = ["qty": quantity]
+
+        if refundTotal != nil {
+            dict["refund_total"] = refundTotal
+        }
+
+        if refundTax != nil {
+            dict["refund_tax"] = Dictionary(uniqueKeysWithValues: refundTax!.map { ($0.taxIDLineItem, $0.amount) })
+        }
+        return dict
+    }
+}
+
+// MARK: - Comparable Conformance
+//
+extension LineItemRefund: Comparable {
+    public static func == (lhs: LineItemRefund, rhs: LineItemRefund) -> Bool {
+        return lhs.quantity == rhs.quantity &&
+            lhs.refundTotal == rhs.refundTotal &&
+            NSDictionary(dictionary: lhs.toDictionary()).isEqual(to: rhs.toDictionary()) &&
```

Comment:
Is it really necessary to compare the dictionaries as well?

Reply:
You are right, in this case, is not needed.